book_src/Operating System From 0 to 1.lyx

```diff
@@ -2643,6 +2644,10 @@ status collapsed
 \end_inset
 
 
+\begin_inset Separator latexpar

```

Comment:
I tried removing them with Emacs, but Lyx still gives a warning about unknown insets. I think it's best to use Lyx 2.1 at the moment. Aside from the warning, the generated PDF looks normal. But I will update to let people use Lyx 2.1.